File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { AdviceSlipApi2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await AdviceSlipApi2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- AdviceSlipApi2 SDK exists test

local sdk = require("advice-slip-api2_sdk")

describe("AdviceSlipApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
